convert2rhel/actions/pre_ponr_changes/subscription.py

```diff
@@ -195,7 +195,15 @@ def run(self):
                     return
                 raise
 
-            logger.warning("No rhsm credentials given to subscribe the system. Did not perform the subscription step.")
+            if subscription._is_registered() and not subscription.is_sca_enabled():
```

Comment:
`_is_registered` is used outside the module, please change it to be `is_registered` to not indicate it's a private function, as it isn't anymore

Reply:
+1